common/common-rest/src/main/java/org/apache/servicecomb/common/rest/codec/param/QueryProcessorCreator.java

```diff
@@ -44,6 +46,13 @@ public Object getValue(HttpServletRequest request) throws Exception {
         value = request.getParameterValues(paramPath);
       } else {
         value = request.getParameter(paramPath);
+        // make some old systems happy
+        if (DynamicPropertyFactory.getInstance()
```

Comment:
every time invoke DynamicPropertyFactory.getInstance().getBooleanProperty, will:
1.create a instance of DynamicBooleanProperty
2.three times map search at least

and this value almost not changed, better to cache it and update by callback.

Reply:
This option do not need make dynamic changeble. I cache it.